Paired t-test is used for comparing two different interventions opplied on the same sample

In [3]:
import pandas as pd
from scipy import stats

In [5]:
#Alchohol cnsumption before and after breakup  
breakups_df = pd.read_csv('breakups.csv')
breakups_df[0:5]

Before_Breakup  After_Breakup
0             470            408
1             354            439
2             496            321
3             351            437
4             349            335

In [4]:
stats.ttest_rel(breakups_df['Before_Breakup'],breakups_df['After_Breakup'])

Ttest_relResult(statistic=-0.5375404241815105, pvalue=0.5971346738292477)

As the p-value is 0.597, which is more than 0.05, we can  conclude that they are part of same distribution.